src/status_im/ui/screens/home/views.cljs

```diff
@@ -44,7 +44,9 @@
       [icons/icon :main-icons/close-circle {:color colors/gray}]]]]
    [react/view
     [react/i18n-text {:style styles/no-chats-text :key :chat-and-transact}]]
-   [invite/button]
+
+   ;; Shivek: Check why this is throwing up...
+   ;; [invite/button]
```

Comment:
?

Reply:
I'll check. This was throwing errors for some reason.